src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,87 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CSV_FILE_SEPERATOR = ",";
+    private static final String OPER_TYPE_SUPPLY = "supply";
+    private static final String OPER_TYPE_BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is public and can be accessed from outside of this class. However, it seems that it is more of a utility method and is not supposed to be a part of the public API of the `WorkWithFile` class. Consider changing the access modifier to `private` or `protected` if you don't expect it to be called from outside of this class or package.